hnn_core/gui.py

```diff
@@ -0,0 +1,1018 @@
+    return sliders
+
+
+def _get_cell_specific_widgets(layout,
+                               style,
+                               location,
+                               data={},
+                               default_data={
+                                   'weights_ampa': {
+                                       'L5_pyramidal': 0.,
+                                       'L2_pyramidal': 0.,
+                                       'L5_basket': 0.,
+                                       'L2_basket': 0.
+                                   },
+                                   'weights_nmda': {
+                                       'L5_pyramidal': 0.,
+                                       'L2_pyramidal': 0.,
+                                       'L5_basket': 0.,
+                                       'L2_basket': 0.
+                                   },
+                                   'delays': {
+                                       'L5_pyramidal': 0.1,
+                                       'L2_pyramidal': 0.1,
+                                       'L5_basket': 0.1,
+                                       'L2_basket': 0.1
+                                   },
+                               }):
+
+    for k in default_data.keys():
+        if k in data:
+            default_data[k].update(data[k])
+
+    kwargs = dict(layout=layout, style=style)
+    cell_types = ['L5_pyramidal', 'L2_pyramidal', 'L5_basket', 'L2_basket']
+    if location == "distal":
+        cell_types.remove('L5_basket')
+
+    weights_ampa, weights_nmda, delays = dict(), dict(), dict()
+    for cell_type in cell_types:
+        weights_ampa[f'{cell_type}'] = FloatText(
+            value=default_data['weights_ampa'][cell_type],
+            description=f'{cell_type}:',
+            **kwargs)
+        weights_nmda[f'{cell_type}'] = FloatText(
+            value=default_data['weights_nmda'][cell_type],
+            description=f'{cell_type}:',
+            **kwargs)
+        delays[f'{cell_type}'] = FloatText(
+            value=default_data['delays'][cell_type],
+            description=f'{cell_type}:',
+            **kwargs)
+
+    widgets_dict = {
+        'weights_ampa': weights_ampa,
+        'weights_nmda': weights_nmda,
+        'delays': delays
+    }
+    widgets_list = ([HTML(value="<b>AMPA weights</b>")] +
+                    list(weights_ampa.values()) +
+                    [HTML(value="<b>NMDA weights</b>")] +
+                    list(weights_nmda.values()) +
+                    [HTML(value="<b>Synaptic delays</b>")] +
+                    list(delays.values()))
+    return widgets_list, widgets_dict
+
+
+def _get_rhythmic_widget(
+    name,
+    tstop_widget,
+    layout,
+    style,
+    location,
+    data={},
+    default_data={
+        'tstart': 0.,
+        'tstart_std': 0.,
+        'tstop': 0.,
+        'burst_rate': 7.5,
+        'burst_std': 0,
+        'repeats': 1,
+        'seedcore': 14,
+    },
```

Comment:
see [comment above](https://github.com/jonescompneurolab/hnn-core/pull/76/files#r913021062) and fix everywhere by using `None` or some such pattern